reference/index.md

```diff
@@ -5,38 +5,148 @@ notoc: true
-|:--------------------------------------------------------------------|:----------------------------------------------------------------|
-| [Dockerfile](/engine/reference/builder/)                            | Defines the contents and startup behavior of a single container |
-| [Compose file](/compose/compose-file/)                              | Defines a multi-container application                           |
-
+<div class="component-container">
+    <!--start row-->
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+             <div class="component-icon">
+                 <a href="/engine/reference/builder/"><img src="/assets/images/build-frontends.svg" alt="Download and install" width="70" height="70"></a>
+                 </div>
+                 <h2 id="dockerfile"><a href="/engine/reference/builder/">Dockerfile</a></h2>
+                <p> Defines the contents and startup behavior of a single container.</p>
+            </div>
+        </div>
+     <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+            <div class="component-icon">
+                <a href="/compose/compose-file/"><img src="/assets/images/build-multi-platform.svg" alt="Release notes" width="70" height="70"></a>
+            </div>
+                <h2 id="compose-file"><a href="/compose/compose-file/">Compose file</a></h2>
+                <p>Defines a multi-container application.</p>
+            </div>
+        </div>
+    </div>
+</div>
 
 ## Command-line interfaces (CLIs)
 
-| CLI                                                            | Description                                                                                |
-|:---------------------------------------------------------------|:-------------------------------------------------------------------------------------------|
-| [Docker CLI](/engine/reference/commandline/cli/)               | The main CLI for Docker, includes all `docker` commands                                    |
-| [Compose CLI](/compose/reference/)                             | The CLI for Docker Compose, which allows you to build and run multi-container applications |
-| [Daemon CLI (dockerd)](/engine/reference/commandline/dockerd/) | Persistent process that manages containers                                                 |
-
+<div class="component-container">
+<!--start row-->
+    <div class="row">
+     <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+        <div class="component">
+            <div class="component-icon">
+                 <a href="/engine/reference/commandline/cli/"><img src="/assets/images/terminal.svg" alt="Docker CLI" width="70" height="70"></a>
+            </div>
+                <h2 id="docker-for-linux"><a href="/engine/reference/commandline/cli/">Docker CLI</a></h2>
+                <p>The main CLI for Docker, includes all `docker` commands.</p>
```

Comment:
`docker` isn't formatted.

Reply:
wrapping "docker" in a `<code>` element should work

Reply:
Woops, thanks for catching that!